code/modules/antagonists/changeling/powers/chameleon_skin.dm

```diff
@@ -11,12 +11,22 @@
 
 /datum/action/changeling/chameleon_skin/sting_action(mob/user)
 	var/mob/living/carbon/human/H = user //SHOULD always be human, because req_human = TRUE
+	if(HAS_TRAIT_FROM(user, TRAIT_SILENT_FOOTSTEPS, CHANGELING_TRAIT))
+		REMOVE_TRAIT(user, TRAIT_SILENT_FOOTSTEPS, CHANGELING_TRAIT)
+	else
+		ADD_TRAIT(user, TRAIT_SILENT_FOOTSTEPS, CHANGELING_TRAIT)
 	if(H.dna.GetSEState(GLOB.chameleonblock))
 		H.dna.SetSEState(GLOB.chameleonblock, 0)
 		singlemutcheck(H, GLOB.chameleonblock, MUTCHK_FORCED)
 	else
 		H.dna.SetSEState(GLOB.chameleonblock, 1)
 		singlemutcheck(H, GLOB.chameleonblock, MUTCHK_FORCED)
+	if(H.dna.GetSEState(GLOB.noprintsblock))
+		H.dna.SetSEState(GLOB.noprintsblock, 0)
+		singlemutcheck(H, GLOB.noprintsblock, MUTCHK_FORCED)
+	else
+		H.dna.SetSEState(GLOB.noprintsblock, 1)
+		singlemutcheck(H, GLOB.noprintsblock, MUTCHK_FORCED)
```

Comment:
We always want these activated and deactivated at the same time, so while it may make less sense to put them all under one conditional, it will make it more consistent
```suggestion
	if(H.dna.GetSEState(GLOB.chameleonblock))
		H.dna.SetSEState(GLOB.chameleonblock, 0)
		singlemutcheck(H, GLOB.chameleonblock, MUTCHK_FORCED)
		H.dna.SetSEState(GLOB.noprintsblock, 0)
		singlemutcheck(H, GLOB.noprintsblock, MUTCHK_FORCED)
		REMOVE_TRAIT(user, TRAIT_SILENT_FOOTSTEPS, CHANGELING_TRAIT)
	else
		H.dna.SetSEState(GLOB.chameleonblock, 1)
		singlemutcheck(H, GLOB.chameleonblock, MUTCHK_FORCED)
		H.dna.SetSEState(GLOB.noprintsblock, 1)
		singlemutcheck(H, GLOB.noprintsblock, MUTCHK_FORCED)
		ADD_TRAIT(user, TRAIT_SILENT_FOOTSTEPS, CHANGELING_TRAIT)
```